src/tests/examples/example_test.py

```diff
@@ -530,6 +546,38 @@ def test_075_kcg_on_four_areas_02(use_ortools, ortools_solver, solver_path):
     check_integrity_first_opt(study_path)
     check_integrity_second_opt(study_path)
 
+# hydro tests
+
+@pytest.mark.short
+def test_hydro_hydro_initialization_1(use_ortools, ortools_solver, solver_path):
+    study_path = ALL_STUDIES_PATH / "short-tests" / "hydro initialization 1"
+    run_study(solver_path, study_path, use_ortools, ortools_solver)
+    reservoir_levels = fetch_hourly_hydro_reservoir_values(study_path)
+    first_january_midnight = 0
+    assert(abs(reservoir_levels[first_january_midnight] - 30) < .05)
+
+@pytest.mark.short
+def test_hydro_hydro_initialization_2(use_ortools, ortools_solver, solver_path):
+    study_path = ALL_STUDIES_PATH / "short-tests" / "hydro initialization 2"
+    run_study(solver_path, study_path, use_ortools, ortools_solver)
+    reservoir_levels = fetch_hourly_hydro_reservoir_values(study_path)
+    first_march_midnight = 1417
+    assert(abs(reservoir_levels[first_march_midnight] - 30) < .05)
+
+@pytest.mark.short
+def test_hydro_hydro_preference_1(use_ortools, ortools_solver, solver_path):
+    study_path = ALL_STUDIES_PATH / "short-tests" / "hydro preference 1"
+    run_study(solver_path, study_path, use_ortools, ortools_solver)
+    reservoir_levels = fetch_hourly_hydro_reservoir_values(study_path)
+    assert(abs(reservoir_levels[-1] - 30.46) < .05)
```

Comment:
What's the behavior of `reservoir_levels[-1]` ? All values are tested ?

Reply:
`array[-1]` refers to the last coefficient of a Python list/array